assets/apps/dashboard/src/Components/Content/AvailableModule.js

```diff
@@ -0,0 +1,210 @@
+/* global neveDash */
+import { __ } from '@wordpress/i18n';
+import {
+	NEVE_AVAIABLE_MODULES_ICON_MAP,
+	NEVE_STORE,
+} from '../../utils/constants';
+import { ArrowRight, LoaderCircle, LucideSettings } from 'lucide-react';
+import Card from '../../Layout/Card';
+import { useState } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import Toggle from '../Common/Toggle';
+import { send } from '../../utils/rest';
+
+const Toast = ({ message }) => {
+	return (
+		<div className="fixed flex items-center gap-2 z-[99999] top-10 right-5 bg-white text-black text-base font-medium px-4 py-2 rounded shadow border border-solid border-blue-600">
+			{message}
+		</div>
+	);
+};
+
+const ModuleToggle = ({
+	slug,
+	moduleData,
+	setMessage,
+	isActive,
+	setIsActive,
+	isInstalled,
+	setIsInstalled,
+}) => {
+	const [loading, setLoading] = useState(false);
+	const { changeModuleStatus, setObfxModuleStatus, setToast } =
+		useDispatch(NEVE_STORE);
+	const { moduleStatus } = useSelect((select) => {
+		const { getObfxModuleStatus } = select(NEVE_STORE);
+
+		return {
+			moduleStatus: getObfxModuleStatus(slug) || false,
+		};
+	});
+
+	const { api } = neveDash;
+	const { title } = moduleData;
+	const toastMessage = {
+		installing: __('Installing Orbix Fox Plugin', 'neve'),
+		activating: __('Activating Orbix Fox Plugin', 'neve'),
```

Comment:
NIT. I think those can be split in `Installing %` & `Activating %` in case we want to re-use the strings when we add a new module.